### PR TITLE
hlmuludiang adds hometown of philadelphia

### PR DESCRIPTION
**Pull request by hlmuludiang**
This pull request is to add Philadelphia to the list of cities

- [ ] Does anyone know of other good cheesesteak places? Not Pats or Genos..

### CODE REVIEW

**Comment by barryburd** [comment]
I was at Pat's yesterday. Yum!

File: hlmuludiang-philadelphia.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Philadelphia, PA
+
+### Restaurants
+- Larry's Steaks
+- Malbec
```
Comment (githubteacher) [inline hlmuludiang-philadelphia.md]: Does Malbec serve 🍷 @hlmuludiang ?

**Comment by barryburd** [comment]
Joe's is pretty good. There's one in Fishtown.

**Comment by hlmuludiang** [comment]
@barryburd Added Pat's & Geno's to the tourist attractions, and Joe's to the local attractions.

**Pull request merged**